plugins/main/vlan/vlan.go

```diff
@@ -152,7 +152,7 @@ func cmdAdd(args *skel.CmdArgs) error {
 	// run the IPAM plugin and get back the config to apply
 	r, err := ipam.ExecAdd(n.IPAM.Type, args.StdinData)
 	if err != nil {
-		return err
+		return fmt.Errorf("failed to execute IPAM delegate: %v", err)
```

Comment:
NIT: It is a good practice to use `%w` instead of `%v` when adding additional context to errors to keep the original error type.
https://golang.org/pkg/fmt/#Errorf

Reply:
hint: check rest of the convention used in the rest of the repo.
Fill welcomed to provide such change globally ;)